guide/onboarding/creating-a-new-wallet.md

```diff
@@ -0,0 +1,91 @@
+
+Within your onboarding process, be clear about what it means to have a non-custodial wallet and the responsibility that they must accept. Within your onboarding process, be clear that your respective team cannot recover "account" information.
+
+{% include /tip/close.html %}
+
+While this may sound daunting to first-time users, these caveats mustn't be hidden. Doing so may severely compromise their funds' safety, as their default assumptions and behaviors could potentially mimic how they typically use centralized applications (for example, feeling like it is OK if they forget their account information because they can contact customer support to help them recover it). According to a study conducted by Ekandari et al about the usability of cryptocurrency wallets, it became clear that users struggled greatly with technical terminology and feeling as though they had a lack of guidance during wallet setup. By walking users through a short set of carousels that clearly frame these Bitcoin-specific features, users will feel well-equipped to confidently navigate and use your wallet.
+
+> Users reported that they often felt like they lacked guidance and understanding during wallet setup
+>
+> <cite>As researched by <a href="https://arxiv.org/pdf/1802.04351.pdf">Eskandari et al</a></cite>
+
+<div class="image-slide-gallery">
+
+{% include picture.html
+   image = "/assets/images/guide/onboarding/creating-a-new-wallet/product-intro-1.png"
+   retina = "/assets/images/guide/onboarding/creating-a-new-wallet/product-intro-1@2x.png"
+   alt-text = "Introduction screen explaining ownership"
+   caption = "Talk about ownership"
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/onboarding/creating-a-new-wallet/product-intro-2.png"
+   retina = "/assets/images/guide/onboarding/creating-a-new-wallet/product-intro-2@2x.png"
+   alt-text = "Introduction screen explaining recovery"
+   caption = "Talk about recovery"
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/onboarding/creating-a-new-wallet/product-intro-3.png"
+   retina = "/assets/images/guide/onboarding/creating-a-new-wallet/product-intro-3@2x.png"
+   alt-text = "Introduction screen explaining security"
+   caption = "Talk about security"
+   width = 250
+   height = 541
+%}
+
+</div>
+
+Above exhibits the typical first screens that users are shown after downloading a Bitcoin wallet, followed by an outline of suggested informational carousels that explicitly lay out/guide users on the unique features of non-custodial wallets to make sure they understand these caveats. These screens suggest talking about ownership of their “account”, explicitly stating the recovery mechanism that the respective wallet uses, and security measures that they should take to protect their wallet.
+
+---
+
+Ok, now let's look at backing up a recovery phrase (coming soon)
```

Comment:
So, this page is actually not about creating a new wallet. It ends with the carousel of info that prepares you for creating a wallet. The actual wallet creation is something the application does in a split second by creating those words, a process the user has no influence on. I don't remember right now if you explain that in the next page, but if not, maybe clarify this here? If not, should the page be called something like "Making new users comfortable"?

Reply:
This is an important point, so at a high level, this page covers the following.

 - Bitcoin wallets being radically different from what a user may be accustomed to in the real world. 
 - Self custody giving full control
 - Bitcoin wallets as keychains 
 - Step by step approach to onboarding 
 - Setting expectations related to self custody e.g "no account recovery"
 - Carousels as a tool for guidance and understanding

I think you are right to say the actual key generation component is missing here so I think it should be included but the more practical nature of this comes in the subsequent sections. In terms of the title of the page, I do see these as crucial steps in creating a new wallet flow even though it doesn't explicitly cover the random number generating process, I will include it though